src/java.base/share/classes/java/lang/ModuleLayer.java

```diff
@@ -297,6 +299,35 @@ public Controller addOpens(Module source, String pn, Module target) {
             source.implAddOpens(pn, target);
             return this;
         }
+
+        /***
+         * Enables native access for a module in the layer if the caller's module
+         * already has native access.
+         *
+         * <p> This method is <a href="foreign/package-summary.html#restricted"><em>restricted</em></a>.
+         * Restricted methods are unsafe, and, if used incorrectly, their use might crash
+         * the JVM or, worse, silently result in memory corruption. Thus, clients should refrain
+         * from depending on restricted methods, and use safe and supported functionalities,
+         * where possible.
+         *
+         * @param target the module that's given native access
```

Comment:
The other methods use the phrase "Updates module ..." so I think target should probably be described here as "The module to update".